### PR TITLE
Rebalances instant stun chems.

### PR DESCRIPTION
**Pull request by GuyonBroadway**
:cl: Guyonbroadway
balance: Neurotoxin (the bar drink) will not instantly stun someone unless delivered in doses of 15u or more. However the longer its in someone the more and more stamina damage they will take , eventually making them collapse from exhaustion.
balance: Beepsky smash no-longer delivers one stun up-front but rather delivers lots of very rapid mini-stuns making it near impossible for anyone to do anything without a little luck. 
/:cl:

[why]: In a syringe with a syringe gun both of these were hard ranged stuns that traveled faster than taser bolts, stunned for a flat six seconds plus however long it was in AND could be loaded with other deadly chems or shit that mutes you. 

Chem has been babied for FAR too long!

_"Oh but GUUUYOOON!!! what if you miss your syringe and have to relaooooood!!! think of the children!!"_   Fuck off, reloading a syringe gun is not fucking hard and keeping a box full of spare reloads is easy as fuck too, plenty of syringe boxes spawn where any proper chem machine is and more can easily be obtained from medbays vending machines or just getting into cargo with a wrench. Plus a scientist can easily get a rapid syringe gun in two to three mins and go fucking ham. You can keep three spare fucking syringes in a pocket protector for crying out loud!

### CODE REVIEW

**Comment by HippieBartels** [comment]
Fuck off, you're trying to nerf this because you lost to it in a round today. We shouldn't make changes based solely on your personal dislike of a certain strat.

Usage of neurotoxin/beepsky smash has never been considered a problem by the playerbase. Nor is it very powerful as it stands.

**Comment by Arzion5** [comment]
We should remove the syringe gun altogether and only allow the traitor one tbh smh

**Comment by Spankmaster1066** [comment]
Idk there's tons of chems like this. Lexorin is a one shot kill. Why nerf? maybe make syringes have a 50% chance to break when fired long range. Im a gay, Im a homosexual? I am not sure where to go from ehere.

File: hippiestation/code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -44,64 +44,64 @@
 /datum/reagent/consumable/ethanol/isoproyl/on_mob_life(mob/living/M)
```
Comment (ghost) [inline hippiestation/code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:44]: what the fuck are all these whitespace changes? Undo all of those

**Review by ghost — DISMISSED** [review]
No

File: hippiestation/code/modules/reagents/chemistry/reagents/alcohol_reagents.dm (hunk 1 above)
Comment (JohnGinnane) [reply hippiestation/code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:44]: it's actually better this way, rather than having trailing spaces

**Comment by GuyonBroadway** [comment]
Leroxin is not an "instant" stun and can be countered by jabbing yerself with an epiniphrine pen to counter the resutling oxyloss.

**Comment by PopNotes** [comment]
yea bretty gud :DD

File: hippiestation/code/modules/reagents/chemistry/reagents/alcohol_reagents.dm (hunk 1 above)
Comment (GuyonBroadway) [reply hippiestation/code/modules/reagents/chemistry/reagents/alcohol_reagents.dm:44]: These are inconsistent as fuck like some of them are just lines of five spaces? You look me in the eye and say they were a good idea.

**Comment by tortellinitony** [comment]
ided pr?

**Comment by GuyonBroadway** [comment]
In my defence I've been meaning to nerf it for some time.

**Comment by tortellinitony** [comment]
so is this why you've nerfed so much stuff, since you get mad you lose to it?

**Comment by GuyonBroadway** [comment]
No, we had a long discussion on stuns in general and thats why I nerfed em.

**Comment by Blute1** [comment]
neurotoxin and beepsky smash are uncommon, unlike taser shots, they should stay in as one of the only instant stuns, if we keep going this route, stamina will take over and the old stun system will be unused

**Comment by Blute1** [comment]
if anything you wont get this passed because of player input, remember your policy, all hippie balance changes need to have extra input from players besides headmins, and i doubt a lot of people want this change

**Comment by Sarcalogo** [comment]
Makes sense. Everything he said is true. Not fair to Security. Oh and by the way, tasers still only have five shots (even though you need double the amount to hit people now).

**Comment by Sarcalogo** [comment]
>They are uncommon
LOL? Are you serious?

**Comment by PopNotes** [comment]
>if we keep going this route, stamina will take over and the old stun system will be unused

good

+2 from me

**Comment by GuyonBroadway** [comment]
Killing the old stuns system is the idea. Bit by bit I will remove one shot stun and ur done gameplay in favour of more interesting mechanics that have more in depth stratergy and counterplay that provide players on both sides of the equation more options to do and to not die.

**Comment by ghost** [comment]
the neurotoxin drink change sounds at least a little cool
but i can't be down with the beepsky smash change, he is the LAW, and should retain his instant stun capabilities.

**Comment by Sarcalogo** [comment]
\>In favour of more interesting mechanics that have more in depth stratergy and counterplay that provide 
When I offered this everyone told me that the players didn't want a deep confusing combat system and that I should fuck off and go play Lifeweb and that melee combat as it is now is a-okay STOP CHANGING THINGSASDGF

**Comment by GuyonBroadway** [comment]
The thing is beepsky smash used to work the way I'm making it work now. It used to spam you with lots of little repeated stuns till someone changed it. This method is far more fun because people will repeatedly drop items and stall moving mid fight as opposed to turning syringe guns into better tasers like it does now.

**Comment by GuyonBroadway** [comment]
Reminder a vote for this PR is a vote to fuck bartels....

and move away from one-stun = done combat.

**Comment by ghost** [comment]
yeah nah but to me if you're gonna make it a series of ministuns just make it a big stun
i get that it gives you .5 seconds to whip out your dick and shoot a guy but ultimately you're still fucked
also it has beepsky in the name
therefore it must provide everyone the opportunity to enforce THE LAW aka pretty much a stun

**Comment by KayeArray** [comment]
still needs more head input

**Comment by KayeArray** [comment]
This is generating a lot of salt -1

**Comment by Carbonhell** [comment]
Imo the beepsky change is fair,it was funnier when you just kept falling down every 2 seconds, whereas the neurotoxin change is bad.

**Comment by PopNotes** [comment]
@banthebantz controversy does not equal a bad change, reconsider

**Comment by ghost** [comment]
i disapprove of this change,  those two chems are barely used tbh, and syringe/chembox is a bulky combination for subpar attack rate and a VERY noticeable inhand sprite.  Plus,  they have to climb the bar table,  etc,  to get it,  chemlab is a public and visible location....

**Comment by Arzion5** [comment]
The mere fact that people use these bar drinks to be able to instantly stun people is not nice. The changes seem pretty good enough, +1.

**Comment by WallaceShark** [comment]
I like the change on Beepsky Smash. Since when has Beepsky been inclined to baton criminals only once?

Dropping a +1.

**Comment by 2Dollarbill** [comment]
-1, I don't see the need to change these. They haven't been much of a problem before, why now.

**Comment by ghost** [comment]
Your PR needs to be updated to reflect the current state of the master branch, please update your PR and either (re)open it, or ask a maintainer to (re)open it.

**Pull request closed without merging**